Another unclear buildbot failure email

I got this buildbot failure email:
```
The Buildbot has detected a new failure on builder lldb-arm-ubuntu while building llvm.

Full details are available at:
    https://lab.llvm.org/buildbot/#/builders/18/builds/150

Worker for this Build: linaro-lldb-arm-ubuntu
Blamelist:
    Jay Foad <jay.foad@amd.com>,
    Jonas Devlieghere <jonas@devlieghere.com>

BUILD FAILED: failed build (failure)

Step 6 (test) failure: build (failure)
...
UNSUPPORTED: lldb-api :: commands/expression/import-std-module/vector-dbg-info-content/TestDbgInfoContentVectorFromStdModule.py (100 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/import-std-module/vector-of-vectors/TestVectorOfVectorsFromStdModule.py (101 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/import-std-module/vector/TestVectorFromStdModule.py (102 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/import-std-module/weak_ptr-dbg-info-content/TestDbgInfoContentWeakPtrFromStdModule.py (103 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/import-std-module/weak_ptr/TestWeakPtrFromStdModule.py (104 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/import_builtin_fileid/TestImportBuiltinFileID.py (105 of 2730)
PASS: lldb-api :: commands/expression/invalid-args/TestInvalidArgsExpression.py (106 of 2730)
PASS: lldb-api :: commands/expression/import_base_class_when_class_has_derived_member/TestImportBaseClassWhenClassHasDerivedMember.py (107 of 2730)
PASS: lldb-api :: commands/expression/inline-namespace/TestInlineNamespace.py (108 of 2730)
PASS: lldb-api :: commands/expression/ir-interpreter-phi-nodes/TestIRInterpreterPHINodes.py (109 of 2730)
XFAIL: lldb-api :: commands/expression/macros/TestMacros.py (110 of 2730)
PASS: lldb-api :: commands/expression/issue_11588/Test11588.py (111 of 2730)
PASS: lldb-api :: commands/expression/ir-interpreter/TestIRInterpreter.py (112 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/multiline-completion/TestMultilineCompletion.py (113 of 2730)
PASS: lldb-api :: commands/expression/expr-in-syscall/TestExpressionInSyscall.py (114 of 2730)
PASS: lldb-api :: commands/expression/memory-allocation/TestMemoryAllocSettings.py (115 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/multiline-navigation/TestMultilineNavigation.py (116 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/namespace_local_var_same_name_cpp_and_c/TestNamespaceLocalVarSameNameCppAndC.py (117 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/namespace_local_var_same_name_obj_c/TestNamespaceLocalVarSameNameObjC.py (118 of 2730)
PASS: lldb-api :: commands/expression/namespace-alias/TestInlineNamespaceAlias.py (119 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/persist_objc_pointeetype/TestPersistObjCPointeeType.py (120 of 2730)
PASS: lldb-api :: commands/expression/options/TestExprOptions.py (121 of 2730)
PASS: lldb-api :: commands/expression/persistent_ptr_update/TestPersistentPtrUpdate.py (122 of 2730)
PASS: lldb-api :: commands/expression/nested/TestNestedExpressions.py (123 of 2730)
PASS: lldb-api :: commands/expression/no-deadlock/TestExprDoesntBlock.py (124 of 2730)
PASS: lldb-api :: commands/expression/persistent_types/TestNestedPersistentTypes.py (125 of 2730)
PASS: lldb-api :: commands/expression/persistent_types/TestPersistentTypes.py (126 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/po_persistent_result/TestPoPersistentResult.py (127 of 2730)
PASS: lldb-api :: commands/expression/persistent_variables/TestPersistentVariables.py (128 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/po_verbosity/TestPoVerbosity.py (129 of 2730)
PASS: lldb-api :: commands/expression/pr52257/TestExprCrash.py (130 of 2730)
PASS: lldb-api :: commands/expression/persistent_result/TestPersistentResult.py (131 of 2730)
PASS: lldb-api :: commands/expression/pr35310/TestExprsBug35310.py (132 of 2730)
PASS: lldb-api :: commands/expression/radar_8638051/Test8638051.py (133 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/rdar44436068/Test128BitsInteger.py (134 of 2730)
PASS: lldb-api :: commands/expression/radar_9531204/TestPrintfAfterUp.py (135 of 2730)
PASS: lldb-api :: commands/expression/radar_9673664/TestExprHelpExamples.py (136 of 2730)
PASS: lldb-api :: commands/expression/rdar42038760/TestScalarURem.py (137 of 2730)
PASS: lldb-api :: commands/expression/regression-access-function-template-in-record/TestRegressionAccessFunctionTemplateInRecord.py (138 of 2730)
PASS: lldb-api :: commands/expression/result_numbering/TestResultNumbering.py (139 of 2730)
PASS: lldb-api :: commands/expression/save_jit_objects/TestSaveJITObjects.py (140 of 2730)
PASS: lldb-api :: commands/expression/scoped_enums/TestScopedEnumType.py (141 of 2730)
PASS: lldb-api :: commands/expression/test/TestExprs2.py (142 of 2730)
PASS: lldb-api :: commands/expression/static-initializers/TestStaticInitializers.py (143 of 2730)
UNSUPPORTED: lldb-api :: commands/expression/two-files/TestObjCTypeQueryFromOtherCompileUnit.py (144 of 2730)
PASS: lldb-api :: commands/expression/timeout/TestCallWithTimeout.py (145 of 2730)
PASS: lldb-api :: commands/expression/unicode-in-variable/TestUnicodeInVariable.py (146 of 2730)
PASS: lldb-api :: commands/expression/test/TestExprs.py (147 of 2730)
PASS: lldb-api :: commands/expression/top-level/TestTopLevelExprs.py (148 of 2730)

Sincerely,
LLVM Buildbot
```
The quoted part of the build log is not useful. It tells me a bunch of tests that did not fail. It gives me no clue as to what did fail or why.

This is similar to #65 but apparently that issue was fixed.

--- a/zorg/buildbot/reporters/utils.py
+++ b/zorg/buildbot/reporters/utils.py
@@ -62,7 +62,7 @@ def get_log_details(build):
                         if line.startswith("o") or line.startswith("e"):
                             lines[j] = line[1:]
                     for j, line in enumerate(lines):
-                        if line.find("FAIL:") != -1 or line.find("FAILED") != -1:
+                        if line.startswith("FAIL:") or line.find("FAILED") != -1:
                             if j > 10:
                                 del lines[:j-10] # Start 10 lines before FAIL
                                 lines = ["..."] + lines

